docker/hq-compose.yml

```diff
@@ -35,7 +35,8 @@ postgres:
     - ${VOLUME_PREFIX}postgresql:/var/lib/postgresql/data
 
 couch:
-  image: klaemo/couchdb:1.6.1
+  image: klaemo/couchdb:2.0-dev
```

Comment:
They moved this to an official apache repo https://github.com/apache/couchdb-docker. haven't looked into moving it yet